### DISCUSSION TITLE
Unable to run OneDriveGUI, The Wizard will not run

### DISCUSSION THREAD

**Discussion by LinuxNovice** [abraunegg/onedrive#2221] [Q&A] [answered]
When I run the OneDriveGUI it won't start the Wizard. Not sure what to do.
I have it working on other systems but occasionally run into this issue.

This is a fresh install of Fedora 37 and everything has been updated.
Does anyone have any ideas?
![OneDrive_Konsole](https://user-images.githubusercontent.com/118029071/202063534-10c95066-34f5-4cc7-bb94-27d8a53ce418.jpg)

**Comment by abraunegg** [comment, marked as answer]
@LinuxNovice 
The GUI is not covered here. Please go to: https://github.com/bpozdena/OneDriveGUI
